extensions/objects/templates/set-of-algebraic-identifier-editor.component.ts

```diff
@@ -42,17 +42,22 @@ interface SetOfAlgebraicIdentifierEditorSchema {
   styleUrls: []
 })
 export class SetOfAlgebraicIdentifierEditorComponent implements OnInit {
-  @Input() modalId: symbol;
-  @Input() value;
+  // These properties are initialized using Angular lifecycle hooks
+  // and we need to do non-null assertion, for more information see
+  // https://github.com/oppia/oppia/wiki/Guide-on-defining-types#ts-7-1
+  @Input() modalId!: symbol;
+  @Input() value!: unknown;
```

Comment:
is it possible to remove the use of unknown here?

Reply:
Argument provided in `updateValue` function is unknown itself. I think this a case like schema-based-editor where we're not really sure what type of values we're receiving